### PR TITLE
2016-12-02 - Node Interactive - Notes

### PR DESCRIPTION
**Pull request by joshgav**
Captures major discussion points and next steps from Diagnostics group meeting after Node Interactive.

@nodejs/diagnostics it was great working together at the summit on Friday. quite an intense discussion which hopefully deepened our shared understandings and goals. we also reached some decisions and concrete next steps in several areas 🎉 . PTAL if the discussion points and next steps captured in these notes are correct.

We could really 🙏  use your help in implementing and/or opening issues accordingly 😄 

Also could you please add to this thread those who were in the meeting but not on the Diagnostics alias? Thanks!

### CODE REVIEW

**Comment by joshgav** [comment]
@sam-github @digitalinfinity @mrkmarron @mhdawson @Trott @jkrems

File: wg-meetings/2016-12-02.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+How can we help the ecosystem with this problem of tracking context across async
+boundaries?
+
+Domains and Zones were attempts.
+
+Could we add a context variable whenever a function is registered? Problem is
+that it's not clear what context the programmer would intend to capture.
+
+async_hooks only works with Node’s native object hierarchy (*Wraps). What about
```
Comment (jkrems) [inline wg-meetings/2016-12-02.md]: Formatting - I think a code tag (`*Wraps`) would fix the leaking italics.

**Comment by jkrems** [comment]
Matches my recollection of events! :)

File: wg-meetings/2016-12-02.md (hunk 1 above)
Comment (watson) [inline wg-meetings/2016-12-02.md]: Or just escape it: `\*`

**Comment by joshgav** [comment]
Landed in https://github.com/nodejs/diagnostics/commit/8dd9bd1147a4461ef069559c45918e614edb7308.

Now to start tackling some of the items identified :)

**Pull request merged**